Nuget package: load opentap bug

When building with the latest beta nuget package, this error occurs when installing packages: `OpenTap.targets(94,5): error MSB4018: System.IO.FileNotFoundException: Could not load file or assembly 'OpenTap, Version=9.4.0.0, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.`

It looks like we are not loading the OpenTAP assembly correctly.

--- a/sdk/OpenTap.Sdk.MSBuild/OpenTapLoader.cs
+++ b/sdk/OpenTap.Sdk.MSBuild/OpenTapLoader.cs
@@ -52,8 +52,8 @@ namespace Keysight.OpenTap.Sdk.MSBuild
                 var openTapDll = Path.Combine(root, "OpenTap.dll");
                 var openTapPackageDll = Path.Combine(root, "OpenTap.Package.dll");
 
-                Assembly.LoadFile(openTapPackageDll);
-                Assembly.LoadFile(openTapDll);
+                Assembly.LoadFrom(openTapPackageDll);
+                Assembly.LoadFrom(openTapDll);
             }
         }
 

